config/environments/test.rb

```diff
@@ -44,4 +44,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Tell Rails to use the system default timezone. This avoids issues where Timecop
+  # freezes to UTC and the fake browser is running under a local timezone
+  config.time_zone = 'America/Los_Angeles'
+  ENV['TZ'] = 'America/Los_Angeles' # Make Capybara aware of the current time zone
```

Comment:
Question for you…is it better to use a fixed time zone (e.g. Los Angeles) than the time zone of the system running the test? If so, why? (I’m not advocating either way, I’m just curious.)

Reply:
Thanks for asking! I had two motivations:

1. I like specs to be explicit so that they don't pass locally and fail on CI. Setting this as configuration achieves this goal. I also intentionally chose a timezone that was not UTC (and not my own) so it would be slightly more obvious when a timezone was not being used either in code or tests. If the app were configured to use UTC and there was a place in the code that didn't reference a timezone, it might accidentally still pass.
2. The [way I found to look up system timezone](https://stackoverflow.com/questions/12445336/local-time-zone-in-ruby) resulted in a string like `EDT` instead of `EST5EDT`, which meant that I couldn't pass it directly to `Time.zone=`. 

Another option would be to require the `TZ` environment variable be set, by adding it to `.env` and to `.travis.yml`. I have no objection to that approach, but this seemed like it would require fewer people to change their local configuration to get the specs running.

Reply:
That makes sense, thanks.